lms/validation/_lti_launch_params.py

```diff
@@ -66,6 +66,22 @@ class _CommonLTILaunchSchema(LTIV11CoreSchema):
     launch_presentation_return_url = fields.Str()
     tool_consumer_info_product_family_code = fields.Str()
 
+    @pre_load
+    def _load_canvas_course_id(self, data, **_kwargs):  # pylint: disable=no-self-use
+        canvas_course_id = data.get("custom_canvas_course_id")
+        if canvas_course_id and not isinstance(canvas_course_id, str):
+            # In LTI1.3 course_id is sent as an integer
+            # instead of as a string in LTI1.1.
+            # Coercing both version to integers could be handled
+            # with fields.Int(strict=False)
+            #
+            # Doing a manual conversion to str here instead so both versions
+            # behave the same as LMS is concern
+            # and to allow for future changes on canvas' side
```

Comment:
What I'm trying to say here at the end is that we rather treat the value as just some opaque value that we forward to the Canvas API.

A string is more opaque than an integer and if they change to let's say "uuids" we are covered.